cmd/jaeger/internal/extension/jaegerstorage/config.go

```diff
@@ -43,6 +45,10 @@ type Backend struct {
 	Opensearch    *esCfg.Configuration    `mapstructure:"opensearch"`
 }
 
+type MetricBackends struct {
+	Prometheus *prometheus.Options `mapstructure:"prometheus"`
```

Comment:
in most other storages we used Configuration, not Options. Options is usually just a wrapper that deals with CLI flags.

Reply:
Ohh okay.